packages/native-stack/src/types.tsx

```diff
@@ -682,6 +708,196 @@ export type NativeStackNavigationOptions = {
    */
   unstable_sheetFooter?: () => React.ReactNode;
 };
+interface SharedHeaderBarButtonItem {
+  /**
+   * Title of the button.
+   */
+  title?: string;
+  /**
+   * Style for the button title.
+   */
+  titleStyle?: {
+    fontFamily?: string;
+    fontSize?: number;
+    fontWeight?: string;
+    color?: ColorValue;
+  };
+  /**
+   * Image source for the button
+   */
+  image?: ImageRequireSource;
```

Comment:
Rename to `imageSource` for consistency with other props.

Reply:
https://github.com/react-navigation/react-navigation/pull/12657/commits/876324db7db6dadb4ea275da44f1a85e89786fe4